### ISSUE TITLE
orbtop should provide immediate error about missing objdump

### ISSUE THREAD

**Issue by zx64** [orbcode/orbuculum#66]
(For clarity, this is tested against b475ebb)

Currently:
```
> objdump.exe
'objdump.exe' is not recognized as an internal or external command,
operable program or batch file.
> echo %OBJDUMP%
%OBJDUMP% (i.e. env var is unset)
> dir /b simple.elf
simple.elf
> orbtop.exe -e simple.elf
(Program launches, only continuously reports "Could not read symbols" error despite elf existing)
```

Expected:
```
> orbtop.exe -e simple.elf
orbtop: Unable to find objdump. Add to path or set with OBJDUMP.
> set OBJDUMP=c:\armgcc\bin\arm-none-eabi-objdump.exe
> orbtop.exe -e simple.elf
(Program works properly)
```

**Comment by Novakov** [comment]
It would be worth to check `objdump` as `--source-comment` option used is available from binutils > 2.33 (released 3 years ago)

**Comment by mubes** [comment]
I'm closing this as fixed. We'll need to re-open it after 2.0.0 to include llvm-objdump to get its speedup goodness, so we'll revisit it all then.
